utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -493,6 +515,7 @@ function Set-DefinitionSection {
             ((-not [String]::IsNullOrEmpty($formattedMaxValue)) ? $formattedMaxValue : $null),
             ((-not [String]::IsNullOrEmpty($formattedRoleNames)) ? $formattedRoleNames : $null),
             ((-not [String]::IsNullOrEmpty($formattedExample)) ? $formattedExample : $null),
+            (($definition.discriminator.propertyName) ? ('- Type-deciding property: {0}' -f $definition.discriminator.propertyName) : $null),
```

Comment:
What about 
```suggestion
            (($definition.discriminator.propertyName) ? ('- Discriminated property: {0}' -f $definition.discriminator.propertyName) : $null),
```
I see that lingo in the docs : https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/data-types#custom-tagged-union-data-type

Reply:
Hey @eriqua,
thanks for the suggestion. I wonder though if `- Discriminating property: {0}` would be better (while following the lingo)?

Reply:
*cough cough* @eriqua *cough cough cough*

Reply:
_cough cough ... cough cough cough_

Reply:
Hey @AlexanderSehr you should really take care of that bad throat!

To the matter at hand: The referenced link explicitly names it "discriminated property" twice. I'm no native but I don't dislike `discriminating` either, just don't like `type-deciding` too much, since `type` is often the discriminator itself.

So `discriminated property`, `discriminating property` or even `discriminator property`/`discriminator` equally work for me

Reply:
Whichever decision above, I'd anyway suggest to also format the property name as follows
```suggestion
            (($definition.discriminator.propertyName) ? ('- Type-deciding property: `{0}`' -f $definition.discriminator.propertyName) : $null),
```

Reply:
Updated to 'discriminator'. I think I originally went with 'type-deciding' instead because it sounded like a more ... explanatory way of explaining it 😄 Also incorporated your suggested change

Reply:
Looks great, thanks!